raster/src/test/scala/geotrellis/raster/io/geotiff/BitGeoTiffMultibandTileSpec.scala

```diff
@@ -87,7 +87,7 @@ class BitGeoTiffMultibandTileSpec extends FunSpec
 
     it("should combine all bands with pixel interleave, striped") {
       val tile =
-        MultibandGeoTiff(p("striped", "pixel")).tile
+        MultibandGeoTiff.decompressed(p("striped", "pixel")).tile
```

Comment:
Not trying to be difficult here but do you think this is not suitable substitute
`MultibandGeoTiff(p("striped", "pixel")).tile.toArrayTile`

Reply:
ah it won't work, `toArrayTile` is in [GeoTiffTile](https://github.com/locationtech/geotrellis/blob/master/raster/src/main/scala/geotrellis/raster/io/geotiff/GeoTiffTile.scala#L690)

Reply:
Ah you're genius and I'm blind!